prerechnung.py

```diff
@@ -1,41 +1,81 @@
+import json
+import os
+
 from api import Api, LIMIT
-import purchase_invoice
+
+from google.cloud import documentai_v1beta3 as documentai
+from google.api_core.client_options import ClientOptions
+
 
 def process(company_name):
-    prs = Api.api.get_list("PreRechnung",filters={'company':company_name,
-                                                  'processed':False},
-                           limit_page_length=LIMIT) 
+    prs = Api.api.get_list(
+        "PreRechnung",
+        filters={'company': company_name, 'processed': False},
+        limit_page_length=LIMIT
+    )
     for pr in prs:
         process_inv(pr)
     print("Prerechnungen vorprozessiert")
-    
+
+
+def extract_invoice_info(pdf_file_content) -> dict:
+    project_id = "quantum-idiom-379621"
+    processor_id = "b9f742465e96697"
```

Comment:
this should not be hard-wired either. It seems that an extra settings field is needed, because this is not part of the Google json.